compiler/testData/codegen/box/traits/abstractClassWithFakeOverride.kt

```diff
@@ -0,0 +1,11 @@
+interface A {
+    fun foo(): String = "OK"
+}
+
+abstract class B : A {
+    abstract override fun foo(): String
+}
+
+abstract class C : B()
+
+fun box() = "OK"
```

Comment:
Let's change the test case a little bit to use the classes from the hierarchy. (If the classes aren't loaded at runtime, they are not verified by the JVM and thus may contain invalid bytecode, which the test wouldn't catch)
```
interface A {
    fun foo(): String = "Fail"
}

abstract class B : A {
    abstract override fun foo(): String
}

abstract class C : B()

class D : C() {
    override fun foo(): String = "OK"
}

fun box() = D().foo()
```